### PR TITLE
config: elasticsearch records doc_type definition

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
ping @jirikuncar

File: invenio_search/config.py [hunk 1]
```diff
@@ -94,3 +94,6 @@
     "980__b": ["collections.secondary"],
     "542__l": ["information_relating_to_copyright_status.copyright_status"],
 }
+
+SEARCH_ELASTICSEARCH_RECORDS_DOC_TYPE = 'record'
```
Comment (jirikuncar) [inline invenio_search/config.py]: Please rename it to `SEARCH_DOCTYPE_DEFAULT = 'record'` and move it after `SEARCH_INDEX_DEFAULT`. Thanks

Comment (hachreak) [inline invenio_search/config.py]: `SEARCH_DOCTYPE_DEFAULT` is not too generic? In this case the configuration  is something strictly correlated with `record`.

**Comment by jirikuncar** [comment]
Superseded by #43.

**Pull request closed without merging**